askem.dbml

```diff
@@ -15,11 +15,29 @@ Enum public.resource_type {
 Enum public.provenance_type {
    Dataset
    Model
+   ModelParameter
    Plan
+   PlanParameter
    ModelRevision
    Intermediate
    Publication
    SimulationRun
+   Project
+   Concept
+}
+
+Enum provenance_abbreviations{
```

Comment:
Should this be in the `askem.dbml`? Isn't this part of the implementation?

Reply:
Couldn't it just be included here: https://github.com/DARPA-ASKEM/data-service/blob/2880f968cb64bb0809faf8b52a12e646227fcd4e/tds/schema/provenance.py#L21-L35